docs/articles/hocon/Index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
```

Comment:
Capitalization - this file will probably need to be lowercase `index.html` since we could be hosting on Linux.

Reply:
Got it!